openquakeplatform/set_thumb/mapthumbnail_test.py

```diff
@@ -14,6 +15,8 @@ def set_thumbs_map_test(self):
 
         pla.get('/maps/')
 
+        time.sleep(5)
+
```

Comment:
are still required this delays ?

Reply:
yes because need to attend the loading of page